<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>firstloginUI</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>673</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Login</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <property name="leftMargin">
     <number>0</number>
    </property>
    <property name="topMargin">
     <number>0</number>
    </property>
    <property name="rightMargin">
     <number>0</number>
    </property>
    <property name="bottomMargin">
     <number>0</number>
    </property>
    <item row="0" column="0">
     <widget class="QScrollArea" name="scrollArea">
      <property name="verticalScrollBarPolicy">
       <enum>Qt::ScrollBarPolicy::ScrollBarAsNeeded</enum>
      </property>
      <property name="widgetResizable">
       <bool>true</bool>
      </property>
      <widget class="QWidget" name="scrollAreaWidgetContents">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>0</y>
         <width>798</width>
         <height>621</height>
        </rect>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout">
        <property name="spacing">
         <number>0</number>
        </property>
        <property name="leftMargin">
         <number>0</number>
        </property>
        <property name="topMargin">
         <number>0</number>
        </property>
        <property name="rightMargin">
         <number>0</number>
        </property>
        <property name="bottomMargin">
         <number>0</number>
        </property>
        <item>
         <widget class="QFrame" name="mainFrame">
          <property name="styleSheet">
           <string notr="true">QFrame#mainFrame {
    background-color: white;
}

QLineEdit {
    border: 2px solid #e2e8f0;
    border-radius: 8px;
    padding: 10px;
    font-size: 10pt;
    font-family: 'Segoe UI', Arial, sans-serif;
    background-color: white;
    color: #333333;
}

QLineEdit:focus {
    border: 2px solid #2563eb;
}

QLineEdit:hover {
    border: 2px solid #cbd5e1;
}

QPushButton#registerButton {
    background-color: #2563eb;
    color: white;
    font-weight: bold;
    font-size: 10pt;
    font-family: 'Segoe UI', Arial, sans-serif;
    border-radius: 8px;
    padding: 10px;
    min-height: 40px;
}

QPushButton#registerButton:hover {
    background-color: #1e3a8a;
}

QPushButton#registerButton:pressed {
    background-color: #1e40af;
}

QPushButton#backButton {
    background-color: transparent;
    color: #64748b;
    font-size: 9pt;
    font-family: 'Segoe UI', Arial, sans-serif;
    border: none;
    padding: 6px;
}

QPushButton#backButton:hover {
    color: #2563eb;
    text-decoration: underline;
}
</string>
          </property>
          <widget class="QLabel" name="titleLabel">
           <property name="geometry">
            <rect>
             <x>200</x>
             <y>30</y>
             <width>400</width>
             <height>35</height>
            </rect>
           </property>
           <property name="text">
            <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-family:'Segoe UI,Arial,sans-serif'; font-size:16pt; font-weight:600; color:#1e3a8a;&quot;&gt;Bem-vindo ao Vinicius&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
           </property>
          </widget>
          <widget class="QLabel" name="logoLabel">
           <property name="geometry">
            <rect>
             <x>300</x>
             <y>75</y>
             <width>200</width>
             <height>100</height>
            </rect>
           </property>
           <property name="text">
            <string/>
           </property>
           <property name="pixmap">
            <pixmap>images/vinil_logo.png</pixmap>
           </property>
           <property name="scaledContents">
            <bool>true</bool>
           </property>
          </widget>
          <widget class="QLabel" name="subtitleLabel">
           <property name="geometry">
            <rect>
             <x>300</x>
             <y>185</y>
             <width>200</width>
             <height>20</height>
            </rect>
           </property>
           <property name="text">
            <string>&lt;html&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-family:'Segoe UI,Arial,sans-serif'; font-size:9pt; color:#2563eb;&quot;&gt;Plataforma de Gerenciamento&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
           </property>
          </widget>
          <widget class="QLabel" name="welcomeLabel">
           <property name="geometry">
            <rect>
             <x>250</x>
             <y>215</y>
             <width>300</width>
             <height>30</height>
            </rect>
           </property>
           <property name="text">
            <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-family:'Segoe UI,Arial,sans-serif'; font-size:13pt; font-weight:600; color:#333333;&quot;&gt;Bem-vindo de Volta!&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
           </property>
          </widget>
          <widget class="QLabel" name="instructionLabel">
           <property name="geometry">
            <rect>
             <x>250</x>
             <y>250</y>
             <width>300</width>
             <height>20</height>
            </rect>
           </property>
           <property name="text">
            <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-family:'Segoe UI,Arial,sans-serif'; font-size:9pt; color:#555555;&quot;&gt;Fa√ßa login para acessar o sistema&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
           </property>
          </widget>
          <widget class="QLabel" name="emailLabel">
           <property name="geometry">
            <rect>
             <x>150</x>
             <y>290</y>
             <width>44</width>
             <height>20</height>
            </rect>
           </property>
           <property name="text">
            <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-family:'Segoe UI,Arial,sans-serif'; font-size:9pt; font-weight:600; color:#333333;&quot;&gt;Email&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
           </property>
          </widget>
          <widget class="QLineEdit" name="emailLineEdit">
           <property name="geometry">
            <rect>
             <x>150</x>
             <y>315</y>
             <width>500</width>
             <height>51</height>
            </rect>
           </property>
           <property name="placeholderText">
            <string>seu@email.com</string>
           </property>
          </widget>
          <widget class="QLabel" name="passwordLabel">
           <property name="geometry">
            <rect>
             <x>150</x>
             <y>370</y>
             <width>81</width>
             <height>20</height>
            </rect>
           </property>
           <property name="text">
            <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-family:'Segoe UI,Arial,sans-serif'; font-size:9pt; font-weight:600; color:#333333;&quot;&gt;Senha&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
           </property>
          </widget>
          <widget class="QLineEdit" name="passwordLineEdit">
           <property name="geometry">
            <rect>
             <x>150</x>
             <y>395</y>
             <width>450</width>
             <height>51</height>
            </rect>
           </property>
           <property name="echoMode">
            <enum>QLineEdit::EchoMode::Normal</enum>
           </property>
           <property name="placeholderText">
            <string>Digite sua senha</string>
           </property>
          </widget>
          <widget class="QToolButton" name="toolButton_3">
           <property name="geometry">
            <rect>
             <x>610</x>
             <y>395</y>
             <width>51</width>
             <height>51</height>
            </rect>
           </property>
           <property name="cursor">
            <cursorShape>PointingHandCursor</cursorShape>
           </property>
           <property name="text">
            <string>üëÅ</string>
           </property>
          </widget>
          <widget class="QPushButton" name="pushButton">
           <property name="geometry">
            <rect>
             <x>310</x>
             <y>470</y>
             <width>181</width>
             <height>51</height>
            </rect>
           </property>
           <property name="styleSheet">
            <string notr="true">background-color: rgb(0, 0, 127);
color: white;
font-weight: bold;
font-size: 11pt;
font-family: 'Segoe UI', Arial, sans-serif;
border-radius: 8px;
padding: 12px;
</string>
           </property>
           <property name="text">
            <string>Entrar</string>
           </property>
          </widget>
          <widget class="QPushButton" name="pushButton_Debug">
           <property name="geometry">
            <rect>
             <x>680</x>
             <y>470</y>
             <width>100</width>
             <height>51</height>
            </rect>
           </property>
           <property name="styleSheet">
            <string notr="true">background-color: rgb(0, 0, 127);
color: white;
font-weight: bold;
font-size: 11pt;
font-family: 'Segoe UI', Arial, sans-serif;
border-radius: 8px;
padding: 12px;
</string>
           </property>
           <property name="text">
            <string>Debug</string>
           </property>
          </widget>
          <widget class="QLabel" name="footerLabel">
           <property name="geometry">
            <rect>
             <x>250</x>
             <y>590</y>
             <width>300</width>
             <height>15</height>
            </rect>
           </property>
           <property name="text">
            <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-family:'Segoe UI,Arial,sans-serif'; font-size:7pt; color:#999999;&quot;&gt;¬© 2025 Vinicius. Todos os direitos reservados.&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
           </property>
          </widget>
          <widget class="QPushButton" name="backButton">
           <property name="geometry">
            <rect>
             <x>260</x>
             <y>540</y>
             <width>300</width>
             <height>30</height>
            </rect>
           </property>
           <property name="cursor">
            <cursorShape>PointingHandCursor</cursorShape>
           </property>
           <property name="text">
            <string>N√¢o tem uma conta? Fa√ßa agora.</string>
           </property>
          </widget>
          <widget class="QPushButton" name="pushButton_Debug_2">
           <property name="geometry">
            <rect>
             <x>20</x>
             <y>470</y>
             <width>100</width>
             <height>51</height>
            </rect>
           </property>
           <property name="styleSheet">
            <string notr="true">background-color: rgb(0, 0, 127);
color: white;
font-weight: bold;
font-size: 11pt;
font-family: 'Segoe UI', Arial, sans-serif;
border-radius: 8px;
padding: 12px;
</string>
           </property>
           <property name="text">
            <string>Ajuda</string>
           </property>
          </widget>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
